solutions/swb-reference/src/SWBStack.ts

```diff
@@ -861,7 +861,7 @@ export class SWBStack extends Stack {
       gsis: [
         {
           indexName: 'getIdentityPermissionsByIdentity',
-          partitionKey: { name: 'Identity', type: AttributeType.STRING },
+          partitionKey: { name: 'identity', type: AttributeType.STRING },
```

Comment:
why this change?

Reply:
This is a part of this [PR](https://github.com/aws-solutions/solution-spark-on-aws/pull/732/files#diff-d52cbf56a77bfc71120b1669d3a69b00dcc221ef47f74d4b7b34cdd0f6d9fc41) and needs to deployed to our test account first as mentioned [here](https://github.com/aws-solutions/solution-spark-on-aws/pull/732#pullrequestreview-1234866718)

Reply:
The comment there makes it seem like the change didnt originate there?

Reply:
The DynamicAuth DynamoDB GSI was created with partitionKey `Identity`, @htztomic pointed out it should have been `identity`. In order to deploy this change we first need to destroy the DDBTable and then recreate it with correct GSI parameters. Hence it would be better to handle this change in a single PR where we update both the SWBStack and the ExampleStack together.

Reply:
gotcha, makes sense

Reply:
Is this going to be a breaking change for anything that has been deployed recently?